src/execution/lineage/pipeline_lineage.cpp

```diff
@@ -60,11 +60,7 @@ void PipelineJoinLineage::AdjustChunkOffsets(idx_t chunk_size, idx_t lineage_idx
 }
 
 idx_t PipelineJoinLineage::GetChildChunkOffset(idx_t lineage_idx) {
-	if (lineage_idx == LINEAGE_BUILD) {
-		return build_child_node->GetChunkOffset();
-	} else {
-		return probe_child_node->GetChunkOffset();
-	}
+	return probe_child_node->GetChunkOffset();
 }
```

Comment:
I think this will still adjust the chunk offset for the BUILD side, it'll just adjust it by the probe child chunk offset. Though I don't think the probe child chunk offset will even be set yet, so maybe this defaults to 0?

Either way, I think maybe something like this would be more clear:
```
if (lineage_idx == LINEAGE_BUILD) {
    return 0; // Don't adjust lineage build offsets
} else {
    return probe_child_node->GetChunkOffset();
}
```